packages/react/src/FrostedGlassVFX/FrostedGlassVFX.module.css

```diff
@@ -0,0 +1,63 @@
+.FrostedGlassVFX {
+  position: relative;
+  background-color: var(--brand-FrostedGlassVFX-bgColor);
+}
+
+.FrostedGlassVFX--rounded-corners {
+  border-radius: var(--brand-borderRadius-xlarge);
+}
+
+@supports (backdrop-filter: blur(var(--brand-FrostedGlassVFX-blurIntensity-low))) or
+  (-webkit-backdrop-filter: blur(var(--brand-FrostedGlassVFX-blurIntensity-low))) {
+  .FrostedGlassVFX--intensity-low {
+    backdrop-filter: blur(var(--brand-FrostedGlassVFX-blurIntensity-low));
+  }
+
+  .FrostedGlassVFX--intensity-medium {
+    backdrop-filter: blur(var(--brand-FrostedGlassVFX-blurIntensity-medium));
+  }
+
+  .FrostedGlassVFX--intensity-high {
+    backdrop-filter: blur(var(--brand-FrostedGlassVFX-blurIntensity-high));
+  }
+}
+
+@supports not (backdrop-filter: blur(var(--brand-FrostedGlassVFX-blurIntensity-low))) {
+  .FrostedGlassVFX--intensity-low::after,
+  .FrostedGlassVFX--intensity-medium::after,
+  .FrostedGlassVFX--intensity-high::after {
+    position: absolute;
+    inset: 0;
+    content: '';
+    background-color: var(--brand-color-canvas-default);
+    z-index: 0;
+    border-radius: var(--brand-borderRadius-xlarge);
+  }
+
+  .FrostedGlassVFX > * {
+    z-index: 1;
+  }
+}
```

Comment:
+1, support looks really good. Also, it looks like the only browsers that don't support `backdrop-filter` also don't support `@supports`, so agree that this can probably be safely removed.

- [Can I use backdrop-filter](https://caniuse.com/css-backdrop-filter)
- [Can I use @supports](https://caniuse.com/?search=@supports)

Reply:
Yeah that's fair. Examples of backdrop-filter seemed to include it but we don't do this in dotcom either, so removing it. Thanks 👍